6_visualize/src/prep_spark_line_fun.R

```diff
@@ -49,10 +49,11 @@ prep_spark_funs_data <- function(storm_data, site_data, timestep_ind, spark_conf
 
     # Stop if there's no data at or before this timestep
     if(nrow(storm_data_i) == 0) {
-      stop(sprintf('no stage data for dateTime<=%s and site=%s', DateTime, site))
-      # warning(sprintf('no stage data for dateTime<=%s and site=%s', DateTime, site))
-      # shapes[[site]] <- NULL
-      # next
+
+      #stop(sprintf('no stage data for dateTime<=%s and site=%s', DateTime, site))
+      warning(sprintf('no stage data for dateTime<=%s and site=%s', DateTime, site))
+      shapes[[site]] <- NULL
+      next
```

Comment:
Maybe for now...but see https://github.com/USGS-VIZLAB/vizstorm-GIF/issues/99#issuecomment-413666567. Is there some way we could push this warning-based check earlier (into 1_fetch) and let it stay a stop() here? Otherwise I think we need to add more logic to gage2spark so that the site points and sparklines match up and the intro animation doesn't break.